suites/apis/requestorTest.js

```diff
@@ -80,3 +80,29 @@ Scenario('Send a revoke request for a policy the user does not have @requestor',
   const revokeResponse = await requestorTasks.createRequestForPolicyID(users.mainAcct.accessTokenHeader, users.user1.username, 'requestor_integration_test', true);
   expect(revokeResponse).to.have.property('status_code', 400);
 });
+
+Scenario('User requests access for a policy with a signed status (and revoke it later) @requestor', async ({
+  users, fence,
+}) => {
+  // Check that the user does not have access to policy 'requestor_integration_test'.
+  let userInfo = await fence.do.getUserInfo(users.user0.accessToken);
+  expect(userInfo.data.authz).to.not.have.property('/requestor_integration_test');
+
+  const { accessTokenHeader } = users.mainAcct;
+  // Create a request for a policy that is present in Arborist
+  const createResponse = await requestorTasks.createRequestForPolicyID(accessTokenHeader, users.user0.username, 'requestor_integration_test', false, 'SIGNED');
+  expect(createResponse).to.have.property('status_code', 201);
```

Comment:
could you add more comments to explain the special case this is testing, for people who are not familiar with requestor